src/transformers/modeling_tf_bert.py

```diff
@@ -303,7 +307,12 @@ def call(self, hidden_states, attention_mask, head_mask, output_attentions, trai
         context_layer = tf.reshape(
             context_layer, (batch_size, -1, self.all_head_size)
         )  # (batch_size, seq_len_q, all_head_size)
-        outputs = (context_layer, attention_probs) if output_attentions else (context_layer,)
+
+        outputs = (context_layer,)
+
+        if tf.executing_eagerly():
```

Comment:
Is it a limitation of TensorFlow that we can't have those statements on the same line?

Reply:
Yes, you have to know if you are in eagerly mode before to do any other condition.